apps/dav/lib/Connector/Sabre/FilesPlugin.php

```diff
@@ -435,14 +437,31 @@ public function handleGetProperties(PropFind $propFind, \Sabre\DAV\INode $node)
 			});
 		}
 
-		if ($node instanceof \OCA\DAV\Connector\Sabre\Directory) {
+		if ($node instanceof Directory) {
 			$propFind->handle(self::SIZE_PROPERTYNAME, function () use ($node) {
 				return $node->getSize();
 			});
 
 			$propFind->handle(self::IS_ENCRYPTED_PROPERTYNAME, function () use ($node) {
 				return $node->getFileInfo()->isEncrypted() ? '1' : '0';
 			});
+
+			$requestProperties = $propFind->getRequestedProperties();
```

Comment:
```suggestion
			$requestProperties = $propFind->getRequestedProperties();
			$nbFiles = null;
			$nbFolders = null;
```

To avoid a potential undefined variable warning